Dockerfile

```diff
@@ -0,0 +1,21 @@
+# multi-stage docker build
+# build stage
+FROM golang:latest AS build-env
+ADD . /src/ch
+ENV GOPATH=/
+
+# check for dep binary so we dont run this
+# check file || curl ....
+#RUN curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+
+RUN cd /src/ch \
+#    && dep ensure \
+    && GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o certificate-host
+
+# final stage
+FROM alpine
```

Comment:
Do we even need alpine here? Can we do a raw binary, or no due to the file we'll have to drop in there?

Reply:
I think since the image needs to have a filesystem (for the files/dirs added below), then it needs an OS.

Reply:
That makes sense, wasn't sure if we could get around that by somehow packaging it in the binary.